test/basic-querying.test.js

```diff
@@ -653,7 +663,7 @@ describe('basic-querying', function() {
       });
     });
 
-    describe('geo queries', function() {
+    bdd.describeIf(connectorCapabilities.geoPoint, 'geo queries', function() {
```

Comment:
Hmm, I believe we are intentionally testing geo queries for all connectors, including those that don't provide native GeoPoint & near query support. IIRC, when the connector does not implement near queries, juggler will perform the search in memory.

Take `db2` and `dashdb` for an example. According to the check on the line 52 above, these connectors don't support GeoPoint type, yet they still pass all `describe('geo queries')` tests.

Let's find a way how to allow the `ibmi` connector pass these test too.

Does the connector support object-values in columns? I think that most of our SQL connectors are storing objects (and arrays) as JSON strings in the database columns. Is this perhaps a missing feature of the `ibmi` connector?